loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -669,6 +644,27 @@ void binary_update_test(void)
  ****************************************************************************/
 void binary_update_aging_test(void)
 {
-	binary_update_myself_test();
+	int ret;
+	binary_update_info_t bin_info;
+
+	while (1) {
+		ret = binary_update_getinfo(APP_NAME, &bin_info);
+		if (ret != OK) {
+			return;
+		}
+
+		/* Copy current binary with a higher version and valid crc. */
```

Comment:
`current binary` is wifiapp. Please change `current binary` to `current micom binary` or something.

Reply:
ok. I will modify it.